### PR TITLE
[NG] Schematics update for using bootstrap's push / pull in Clarity grid

### PR DESCRIPTION
**Pull request by jeeyun**
Screenshot of a sample run:
screen shot 2018-08-22 at 3 46 29 pm

Signed-off-by: Jeeyun Lim <jeeyun.lim@gmail.com>

### CODE REVIEW

File: src/schematics/src/update/update-0.13.0.ts [hunk 1]
```diff
@@ -76,6 +76,16 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Alert to using bootstrap's push / pull in Clarity grid
+          if (updated.search(/push-(xs|sm|md|lg|xl)-([0-9])/g) > -1) {
```
Comment (gnomeontherun) [inline src/schematics/src/update/update-0.13.0.ts]: We also need a check for `pull-` classes.

Comment (jeeyun) [reply src/schematics/src/update/update-0.13.0.ts]: doh.. you are right!

File: src/schematics/src/update/update-0.13.0.ts [hunk 2]
```diff
@@ -76,6 +76,16 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Alert to using bootstrap's push / pull in Clarity grid
+          if (updated.search(/(push|pull)-(xs|sm|md|lg|xl)-([0-9])/g) > -1) {
```
[the remark this answers is not in the document]
Comment (gnomeontherun) [reply src/schematics/src/update/update-0.13.0.ts]: `search()` returns the index, or -1, of the first match it finds in the string. So using `> -1` is a way to detect presence of the `pull-sm-5` and `push-md-4` type classes and alert based on that.

[the remark this answers is not in the document]
Comment (jeeyun) [reply src/schematics/src/update/update-0.13.0.ts]: I believe the search returns the index of the first match (of the whole regex expression and not of the subgroups' matches).

[the remark this answers is not in the document]
Comment (hippee-lee) [reply src/schematics/src/update/update-0.13.0.ts]: Thanks!

**Pull request merged**